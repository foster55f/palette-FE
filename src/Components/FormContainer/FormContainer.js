import React from 'react';
import './FormContainer.scss';
import ProjectForm from '../ProjectForm/ProjectForm';
import PaletteForm from '../PaletteForm/PaletteForm'

export const FormContainer = () => {
  return (
    <> 
      <ProjectForm />
      <PaletteForm />
    </>
  )
}

export default FormContainer;