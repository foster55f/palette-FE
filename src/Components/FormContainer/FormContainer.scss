@import '../../scss/variables';

.main-box {
  border-radius: 30px;
  background-color: $background-neutral-color;
  width: 60%;
  margin: 0 auto;
  padding: 2em;
}