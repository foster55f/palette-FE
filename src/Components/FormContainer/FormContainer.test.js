import React from 'react';
import { shallow } from 'enzyme';
import FormContainer from './FormContainer';

describe('FormContainer', () => {
 
it('should match the FormContainer snapshot', () => {
    let wrapper = shallow(<FormContainer />);
    expect(wrapper).toMatchSnapshot();
  })
})