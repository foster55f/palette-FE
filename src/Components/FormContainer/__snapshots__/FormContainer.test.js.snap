// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer should match the FormContainer snapshot 1`] = `
<article
  className="main-box"
>
  <ProjectForm />
  <PaletteForm />
</article>
`;
